tests/e2e/ctl_v3_member_test.go

```diff
@@ -55,6 +59,92 @@ func TestCtlV3MemberUpdatePeerTLS(t *testing.T) {
 	testCtl(t, memberUpdateTest, withCfg(*e2e.NewConfigPeerTLS()))
 }
 
+// TestCtlV3ConsistentMemberList requires the gofailpoint to be enabled.
+// If you execute this case locally, please do not forget to execute
+// `make gofail-enable`.
+func TestCtlV3ConsistentMemberList(t *testing.T) {
+	e2e.BeforeTest(t)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
I get timeout on 20 seconds. Can we just skip this timeout?

Reply:
resolved & thx